wski18n/resources/en_US.all.json

```diff
@@ -21,7 +21,7 @@
   },
   {
     "id": "msg_cmd_desc_long_root",
-    "translation": "The OpenWhisk Deployment Tool\n\nTo begin working with wskdeploy, run the 'wskdeploy' command:\n\n\t$ wskdeploy\n\nThis will deploy OpenWhisk assets specified in manifest.yaml or manifest.yml\nIf current directory doesnt have any manifest file, wskdeploy prints this help message.\n\nThe most common ways of using wskdeploy to deploy and/or undeploy OpenWhisk assets are:\n\n$ wskdeploy\n$ wskdeploy -m path/to/manifest.yaml\n$ wskdeploy -m path/to/manifest.yaml -d path/to/deployment.yaml\n$ wskdeploy undeploy\n$ wskdeploy undeploy -m path/to/manifest.yaml\n$ wskdeploy undeploy -m path/to/manifest.yaml -d path/to/deployment.yaml\n\nAunthentication:\nwskdeploy reads credentials from $HOME/.wskprops by default\nOverwrite default config file on CLI:\n  $ wskdeploy --config <config file> -m path/to/manifest.yaml\nOr specify all three on CLI:\n  $ wskdeploy --apihost HOST --auth KEY --namespace NAMESPACE -m path/to/manifest.yaml"
```

Comment:
These changes need to be "built" using the go-bindata tool to render them as ascii-encoded data using the following instructions: https://github.com/apache/openwhisk-wskdeploy/blob/master/wski18n/README.md